k8s/Makefile

```diff
@@ -25,7 +25,7 @@ create-helm-charts:
 
 .PHONY: create-yaml
 create-yaml:
-	helm template seldon-core-v2-certs ./helm-charts/seldon-core-v2-certs | grep -v "namespace:" > yaml/certs.yaml
+	helm template -n seldon-mesh seldon-core-v2-certs ./helm-charts/seldon-core-v2-certs | grep -v "namespace:" > yaml/certs.yaml
```

Comment:
Concretely, I don't know. However, the _currently generated_ certs helm chart (and the .yaml files) have seldon-mesh, so not adding the namespace here would have changed lots of dnsNames from `seldon-mesh` to `default` on re-generation. I've simply added this here to avoid _any_ changes to the generated certs.yaml

Reply:
So, I've checked. In all the other ones `{{ .Release.Namespace }}` only appears as a value of the `namespace` key, which is removed by the `grep -v`, so the resulting yamls do not end up containing any hardcoded values. In that sense they are generic.